connector/fyle_integrations_platform_connector/apis/expenses.py

```diff
@@ -219,3 +219,10 @@ def post_bulk_accounting_export_summary(self, data: List[Dict]):
             'data': data
         }
         return self.connection.post_bulk_accounting_export_summary(payload)
+
+    def construct_expense_object(self, data: List[Dict], workspace_id: int):
+        """
+        Convert the fyle expenses to the integration expenses schema
+        """
+        self.set_workspace_id(workspace_id=workspace_id)
```

Comment:
why do we need this?

Reply:
attribute_is_valid method in base class requires workspace